src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -366,27 +366,25 @@ impl<S: StateStore> StorageTable<S> {
         Ok(iter)
     }
 
-    /// Iterates on the table with the given prefix of the pk in `pk_prefix` and the range bounds of
-    /// the next primary key column in `next_col_bounds`.
-    // TODO: support multiple datums or `Row` for `next_col_bounds`.
+    /// Iterates on the table with the given prefix of the pk in `pk_prefix` and the range bounds.
     async fn iter_with_pk_bounds(
```

Comment:
Does this function ensures results to be ordered by PK? It is not implied in the definition but I think your `current_row` relies on it.

Reply:
It contains a parameter `ordered` which can guarantee its output is ordered by PK.

Reply:
Oops, I missed that.